code/modules/spells/spell_types/touch_attacks.dm

```diff
@@ -81,3 +81,21 @@
 
 	action_icon_state = "statue"
 	sound = 'sound/magic/fleshtostone.ogg'
+
+/obj/effect/proc_holder/spell/targeted/touch/honk
+	name = "Curse of the Clown"
+	desc = "This spell charges your hand with the vengeful anger of a thousand clowns, turning victims into evil clumsy cluwnes."
+	hand_path = /obj/item/melee/touch_attack/honk
+
+	school = "transmutation"
+	charge_max = 800 // you have to wait 1 minute and 20 seconds because you're dropping all items and turning your target into a clown
+	clothes_req = TRUE
+	cooldown_min = 400 // 100 deciseconds reduction per rank
```

Comment:
isnt 800 - 500 300

Reply:
I am bad at math

Reply:
Actually, can't you upgrade spells only 4 times?
800 - 100 - 100 - 100 - 100 = 400

Reply:
what trollbreeder said